lib/tasks/setup.rake

```diff
@@ -1,13 +1,25 @@
 namespace :setup do
 
-  task :run, [:environment] do
+  task :migrate, [:environment] do
```

Comment:
Why are we renaming all of these tasks? Maybe we should just put them all under the same namespace.